test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/LegacyCSProjPackageReferenceProjectTests.cs

```diff
@@ -47,6 +46,25 @@ public class LegacyCSProjPackageReferenceProjectTests
         }
 
         [Fact]
+        public async Task LCPRP_ThrowsUnavailableExceptionWhenThereIsNoBaseIntermediateOutputPath()
```

Comment:
`LCPRP_WhenThereIsNoBaseIntermediateOutputPath_ThrowsUnavailableException`

Does that seem okay?